advanced_filters/forms.py

```diff
@@ -134,7 +140,7 @@ def _parse_query_dict(query_data, model):
                 # this is a date/datetime field
                 query_data['operator'] = "range"  # default
             else:
-                query_data['operator'] = "iexact"  # default
+                query_data['operator'] = operator  # default
```

Comment:
Interesting, I don't believe I  was not aware of this issue. Do you mind adding a unit test for this method?